texdoc/__init__.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+"""texdoc extension
+
+This is an extension to search for LaTeX documentation.
+
+Synopsis: <trigger> <query>"""
+
+import os
+import re
+import subprocess
+
+from shutil import which
+from albertv0 import *
+
+__iid__          = "PythonInterface/v0.3"
+__prettyname__   = "TeXdoc"
+__version__      = "1.0"
+__trigger__      = "td"
+__author__       = "Florian Adamsky"
+__dependencies__ = ["texdoc"]
+
+iconPath   = os.path.dirname(__file__)+"/texdoc-logo.svg"
+texdoc_cmd = ["texdoc", "-I", "-q", "-s", "-M"]
+
+if which("texdoc") is None:
+    raise Exception("'texdoc' is not in $PATH.")
+
+def handleQuery(query):
+    if not query.isTriggered:
+        return
+
+    query.disableSort()
```

Comment:
I believe the `disableSort` method is not available in Albert v0.2 version. I'm using something like this in my plugins to the same effect:

```
# be backwards compatible with v0.2
if "disableSort" in dir(query):
    query.disableSort()
```

Reply:
v0.2 is incredibly old (6 years, give or take); is there really any benefit to adding complexity to support such an outdated version? If anyone *is* still using it, they're long overdue for an update ;)

Reply:
@dshoreman bergercookie is talking of the python interface version. v0.3 is not even documented. disablesort is the only change to the interface at all. (see https://github.com/albertlauncher/plugins/blob/master/python/src/extension.cpp).

@bergercookie note that theres nothing like backwards compatibility as long as all the versions are below 1. Albert is still beta.

Reply:
> @bergercookie note that theres nothing like backwards compatibility as long as all the versions are below 1. Albert is still beta.

sure, I understand, but I think that given it's a small effort to keep compatible with both, to have this sort of check before calling `.disableSort()`

Reply:
> bergercookie is talking of the python interface version. v0.3 is not even documented. 
Aha, that makes much more sense! Thanks for the link, if it hasn't been done already then I might create an issue later to get that method added in docs (unless I have time to just PR).

---

I'm on the fence here. After some digging, `disableSort` was added back in Jan 2019 (so anyone building from git will have it) but it wasn't officially released until a few weeks ago. I'm inclined to side with @ManuelSchneid3r and go the semver route, ignoring old versions since it's pre-1.0. Presumably there's a chance a lot of people are still on 0.16.1 if they're using outdated distro repositories? But in that case they probably won't have updated plugins either... Maybe? :joy: 

'scuse the essay